packages/botonic-cli/src/botonic-playground-service.ts

```diff
@@ -24,7 +36,10 @@ export class PlaygroundService {
 
   async start() {
     try {
-      this.tunnel = await localtunnel({ port: this.port })
+      this.tunnel = await withTimeout<localtunnel.Tunnel>(
+        localtunnel({ port: this.port }),
+        'Tunneling service not reachable.'
+      )
```

Comment:
Maybe 2 seconds are not enough for `localtunnel` to be executed? How about increasing a little bit the timeout?

Reply:
I've tested it with 1 second (now localtunnel works again) works pretty good. Leaving it to 2 seconds, it should be more than enough.